app/controllers/api/projects.php

```diff
@@ -31,14 +31,6 @@
     }
 }, ['project'], 'projects');
 
-App::init(function ($project) {
-    /** @var Utopia\Database\Document $project */
-
-    if($project->getId() !== 'console') {
-        throw new Exception('Access to this API is forbidden.', 401);
```

Comment:
Why do we remove this security measure that we just added?

Reply:
It was there twice